powerauth-webflow-authentication/src/main/java/io/getlime/security/powerauth/lib/webflow/authentication/repository/model/entity/OperationSessionEntity.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2017 Lime - HighTech Solutions s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.getlime.security.powerauth.lib.webflow.authentication.repository.model.entity;
+
+import io.getlime.security.powerauth.lib.nextstep.model.enumeration.AuthResult;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Date;
+import java.util.Objects;
+
+/**
+ * Entity which stores mapping of operations to HTTP sessions and operation results.
+ *
+ * @author Roman Strobl, roman.strobl@lime-company.eu
+ */
+@Entity
+@Table(name = "wf_operation_session")
+@NamedQueries({
+        @NamedQuery(name = "OperationSessionEntity.findActiveOperationsByHttpSessionId", query = "SELECT o FROM OperationSessionEntity o WHERE o.httpSessionId = ?1 AND o.result='CONTINUE'")
```

Comment:
Rather question than a comment - what do we get by using NamedQuery atop of the Entity, not inside the Repository implementation?

Reply:
I've seen it in articles and tutorials done like this. I don't feel strongly about this issue, we can move it to Repository in next release. It is actually quite nice to see the NamedQuery right next to the method in Repository, so I like your suggestion.